scripts/reporting/issues.py

```diff
@@ -108,7 +109,13 @@ def __init__(self, verbose, issue, metadata):
         first_field = self.body["requestError"].split('","')[1]
         field_row = metadata[metadata["field_name"] == first_field]
         self.form = field_row["form_name"].item()
-        instrument = self.body["experiment_site_id"].split("-")[0]
+        experiment_site_id_regex = (
+            f"({utils.STUDY_ID_REGEX}_{utils.EVENT_REGEX}_)?(.*)-"
```

Comment:
@gcyoung1 can you explain this a bit to me (or at least what exactly you're trying to achieve)?

This will match `"garbage-"` or `"studyid_event_garbage-"`.

Reply:
I'm trying to extract the form. Sometimes this is the only thing in the experiment_site_id before the -, as in [this](https://github.com/sibis-platform/ncanda-operations/issues/10807) issue. Other times there is a studyid_event_ coming before, as in [this](https://github.com/sibis-platform/ncanda-operations/issues/10802) issue. The reason I'm using .* is because I'm not sure how to make a regex for the form names.
It might be simpler to just extract whatever is between the last _, if one exists, and the -.